python_modules/dagster/dagster/core/host_representation/origin.py

```diff
@@ -143,16 +144,37 @@ def create_location(self) -> NoReturn:
 
 @whitelist_for_serdes
 class InProcessRepositoryLocationOrigin(
-    NamedTuple("_InProcessRepositoryLocationOrigin", [("recon_repo", ReconstructableRepository)]),
+    NamedTuple(
+        "_InProcessRepositoryLocationOrigin",
+        [
+            ("loadable_target_origin", LoadableTargetOrigin),
+            ("container_image", Optional[str]),
+            ("entry_point", List[str]),
+        ],
+    ),
```

Comment:
are you confident its safe to change this serdes object? is it effectively just used for x-proc comms in test?

Reply:
Yeah, nobody is persisting these origins outside of tests (https://github.com/dagster-io/internal/pull/1588/ adds a non-test use case that uses an InProcessRepositoryLocationOrigin to load repository data, but we don't include that origin in the data that is subsequently persisted)